test/acceptance/open_and_close.robot

```diff
@@ -63,3 +63,29 @@ Closing all browsers clears cache
     Run Keyword And Expect Error
     ...    No browser with index or alias 'Browser 2' found.
     ...    Switch Browser    Browser 2
+
+Click Element Without Modifier
+    Open Browser    ${ROOT}/jquery.html    ${BROWSER}
+    Set Selenium Implicit Wait    60s
+    Click Element    link=Link with id
+    Sleep    5s
```

Comment:
Adding plains sleep is not good idea and because #757 is not implemented please do something like this: 
``` 
| Wait Until Keyword Succeeds | 10 | 1 | Location Should Be  | ${ROOT}/broken.html |
```

Fix in all places where `Sleep` keyword is used.

Reply:
Remove it